tests/unit/tests_high_five_base.cpp

```diff
@@ -2453,58 +2443,61 @@ void check_multiple_string(Proxy proxy, size_t string_length) {
+        auto obj = CreateTraits::create(file, "underlength_nullterm", dataspace, datatype);
+        REQUIRE_THROWS(obj.write(value));
     }
 
     SECTION("underlength nullpad") {
         auto datatype = FixedLengthStringType(string_length - 1, StringPadding::NullPadded);
-        REQUIRE_THROWS(proxy.create("underlength_nullpad", dataspace, datatype).write(value));
+        auto obj = CreateTraits::create(file, "underlength_nullpad", dataspace, datatype);
+        REQUIRE_THROWS(obj.write(value));
     }
 
     SECTION("underlength spacepad") {
         auto datatype = FixedLengthStringType(string_length - 1, StringPadding::NullTerminated);
-        REQUIRE_THROWS(proxy.create("underlength_spacepad", dataspace, datatype).write(value));
+        auto obj = CreateTraits::create(file, "underlength_spacepad", dataspace, datatype);
+        REQUIRE_THROWS(obj.write(value));
     }
 }
 
 TEST_CASE("HighFiveSTDString (dataset, single, short)") {
     File file("std_string_dataset_single_short.h5", File::Truncate);
-    check_single_string(ForwardToDataSet(file), 3);
+    check_single_string<testing::DataSetCreateTraits>(file, 3);
 }
 
 TEST_CASE("HighFiveSTDString (attribute, single, short)") {
     File file("std_string_attribute_single_short.h5", File::Truncate);
-    check_single_string(ForwardToAttribute(file), 3);
+    check_single_string<testing::AttributeCreateTraits>(file, 3);
 }
 
 TEST_CASE("HighFiveSTDString (dataset, single, long)") {
     File file("std_string_dataset_single_long.h5", File::Truncate);
-    check_single_string(ForwardToDataSet(file), 256);
+    check_single_string<testing::DataSetCreateTraits>(file, 256);
 }
 
 TEST_CASE("HighFiveSTDString (attribute, single, long)") {
     File file("std_string_attribute_single_long.h5", File::Truncate);
-    check_single_string(ForwardToAttribute(file), 256);
+    check_single_string<testing::AttributeCreateTraits>(file, 256);
 }
 
 TEST_CASE("HighFiveSTDString (dataset, multiple, short)") {
     File file("std_string_dataset_multiple_short.h5", File::Truncate);
-    check_multiple_string(ForwardToDataSet(file), 3);
+    check_multiple_string<testing::DataSetCreateTraits>(file, 3);
 }
 
 TEST_CASE("HighFiveSTDString (attribute, multiple, short)") {
     File file("std_string_attribute_multiple_short.h5", File::Truncate);
-    check_multiple_string(ForwardToAttribute(file), 3);
+    check_multiple_string<testing::AttributeCreateTraits>(file, 3);
 }
 
 TEST_CASE("HighFiveSTDString (dataset, multiple, long)") {
     File file("std_string_dataset_multiple_long.h5", File::Truncate);
-    check_multiple_string(ForwardToDataSet(file), 256);
+    check_multiple_string<testing::DataSetCreateTraits>(file, 256);
 }
 
 TEST_CASE("HighFiveSTDString (attribute, multiple, long)") {
     File file("std_string_attribute_multiple_long.h5", File::Truncate);
-    check_multiple_string(ForwardToAttribute(file), 256);
```

Comment:
Seems like `ForwardToAttribute` and `ForwardToDataSet` are not actually removed in this PR? Lines 2266 and following...

Reply:
Thanks, it's gone now.